cas-management-webapp-support/src/main/java/org/jasig/cas/config/CasManagementWebAppConfiguration.java

```diff
@@ -43,6 +43,7 @@
  * @since 4.3.0
  */
 @Configuration("casManagementWebAppConfiguration")
+@Lazy(true)
```

Comment:
Yes for sure. The mgmt webapp is likely not a good example (I only kept it for consistent), but for the webapp, it spins up much faster than before. I also would say that scanning xml context is much slower than reading binary compiled configuration code. So the fact that code is now handled via Java config and less via XML might also contribute to faster startup time.